deploy/csi-rclone/values.yaml

```diff
@@ -1,37 +1,19 @@
+storageClassName: csi-rclone 
 namespace:
   enabled: true
```

Comment:
I dont understand why we need this. `helm install` has a `--create-namespace` option that will crate the namespace you are trying to install in if it des not exist. The helm chart should not be responsible for making the namespace - that is up to the admin or whoever is installing the chart.

Reply:
But it is ok we can clean this up in a follow up PR. It is just confusing to have the helm chart make this.

Reply:
I think this is just some leftover from some fork